crux-llvm/test-data/golden/fneg.c

```diff
@@ -0,0 +1,10 @@
+#include <crucible.h>
+
+double negate(double x) {
+  return -x;
+}
+
+int main() {
+  double x = crucible_double("x");
+  return (x > 0);
```

Comment:
This test case doesn't actually call `negate`.  It probably should.

Reply:
Well that's embarrassing. Fixed.